static/js/components/PersonalTab.js

```diff
@@ -25,86 +24,80 @@ import type {
   AvailableProgram,
   AvailablePrograms,
 } from '../flow/enrollmentTypes';
-import type { Event } from '../flow/eventType';
 import {  validationErrorSelector } from '../util/util';
+import type { Option } from '../flow/generalTypes';
 
 export default class PersonalTab extends React.Component {
   props: {
-    profile:        Profile,
-    errors:         ValidationErrors,
-    saveProfile:    SaveProfileFunc,
-    updateProfile:  UpdateProfileFunc,
-    ui:             UIState,
-    nextStep:       () => void,
-    prevStep:       () => void,
-    programs:       AvailablePrograms,
-    setProgram:     Function,
-    addProgramEnrollment: Function,
+    addProgramEnrollment:     Function,
     currentProgramEnrollment: AvailableProgram,
+    errors:                   ValidationErrors,
+    nextStep:                 () => void,
+    prevStep:                 () => void,
+    profile:                  Profile,
+    programs:                 AvailablePrograms,
+    saveProfile:              SaveProfileFunc,
+    setProgram:               Function,
+    ui:                       UIState,
+    updateProfile:            UpdateProfileFunc,
   };
 
-  programListing = (programs: AvailablePrograms) => {
-    const makeMenuItems = R.map(program => (
-      <MenuItem value={program.id} key={program.id} primaryText={program.title} />
-    ));
-    const sortPrograms = R.sortBy(R.compose(R.toLower, R.prop('title')));
-    const menuItems = R.compose(makeMenuItems, sortPrograms);
-    return menuItems(programs);
-  };
+  sortPrograms = R.sortBy(R.compose(R.toLower, R.prop('title')));
+
+  programOptions = R.compose(
+    R.map(program => ({value: program.id, label: program.title})), this.sortPrograms
+  );
 
-  setProgramHelper = (event: Event, key: string, value: string) => {
+  onChange = (selection: Option): void => {
     const {
       programs,
       setProgram,
     } = this.props;
-    let selected = programs.find(program => program.id === parseInt(value));
-    setProgram(selected);
+    if ( selection && selection.value ) {
+      let selected = programs.find(program => program.id === parseInt(selection.value));
+      setProgram(selected);
+    } else {
+      setProgram("");
```

Comment:
Can you change this to `setProgram(null)`? The type expected here is `AvailableProgram` I think so I'm not sure an empty string fits here. Can you also update the type in the props?